debian/tests/upstream

```diff
@@ -24,48 +24,48 @@ echo "using tmpdir: $WORKDIR/tmp"
 
 echo "Setting up skip-tests-list"
 
-# Use unstable-tests list as base to skip all tests  considered unstable
-# or create an empty file if that upstream file does not exists on this branch
-cp /usr/share/mysql/mysql-test/unstable-tests $SKIP_TEST_LST || touch $SKIP_TEST_LST
-
-# Also use the arch specific skiplists if exist
-if [ -f /usr/share/mysql/mysql-test/unstable-tests.$ARCH ]
+# Use the arch specific skiplists if exist, otherwise list is empty
+if [ -f "/usr/share/mysql/mysql-test/unstable-tests.$ARCH" ]
 then
-  cat /usr/share/mysql/mysql-test/unstable-tests.$ARCH >> $SKIP_TEST_LST
+  cat "/usr/share/mysql/mysql-test/unstable-tests.$ARCH" >> "$MTR_SKIP_TEST_LIST"
 fi
 
 # Skip tests that cannot run properly on ci.debian.net / autopkgtests.ubuntu.com
-cat >> $SKIP_TEST_LST << EOF
+cat >> "$MTR_SKIP_TEST_LIST" << EOF
 binlog.binlog_server_start_options : Requires writable /usr
 main.ctype_uca : Requires writable /usr
 rpl.rpl_gtid_mode : Requires starting server as root ref http://bugs.mysql.com/bug.php?id=70517
 EOF
 
 # Skip tests that cannot run properly on Gitlab-CI
-if [ ! -z "$GITLAB_CI" ]
+if [ -n "$GITLAB_CI" ]
 then
-  cat >> $SKIP_TEST_LST << EOF
+  cat >> "$MTR_SKIP_TEST_LIST" << EOF
 main.mysqld--help : For unknown reason table-cache is 4000 instead of default 421
 EOF
 fi
 
 if [ "$ARCH" = "s390x" ]
 then
-  echo "main.func_regexp_pcre : recursion fails on s390x https://bugs.launchpad.net/ubuntu/+source/mariadb-10.1/+bug/1723947" >> $SKIP_TEST_LST
+  echo "main.func_regexp_pcre : recursion fails on s390x https://bugs.launchpad.net/ubuntu/+source/mariadb-10.1/+bug/1723947" >> "$MTR_SKIP_TEST_LIST"
 elif [ "$ARCH" = "armhf" ] || [ "$ARCH" = "i386" ]
 then
-  echo "main.failed_auth_unixsocket : Test returns wrong exit code on armhf and i386 (but only in debci) https://jira.mariadb.org/browse/MDEV-23933" >> $SKIP_TEST_LST
+  echo "main.failed_auth_unixsocket : Test returns wrong exit code on armhf and i386 (but only in debci) https://jira.mariadb.org/browse/MDEV-23933" >> "$MTR_SKIP_TEST_LIST"
```

Comment:
[MDEV-23933](https://jira.mariadb.org/browse/MDEV-23933) is fixed.

Reply:
So this can be removed altogether?

Reply:
yes